src/app.js

```diff
@@ -0,0 +1,16 @@
+
+import express from "express";
+const app=express();
+import routes from "./routes/index";
+import welcomeRoute from "./routes/welcome"
+
+app.use(express.json());
+app.use(routes)
+
+app.use("/welcomes",welcomeRoute);
```

Comment:
you shouldn't be having this in the app.js file
it should be in the routes folder